ui_framework/components/pager/pager_buttons.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { KuiButton, KuiButtonIcon, KuiButtonGroup } from '../button';
+
+export function KuiPagerButtons({ className, onPrevious, onNext, hasNext, hasPrevious, ...rest }) {
+  return (
+    <KuiButtonGroup isUnited className={className} { ...rest }>
+      <KuiButton
+        data-test-subj="pagerPreviousButton"
+        type="basic"
+        onClick={onPrevious}
+        disabled={!hasPrevious}
+        icon={<KuiButtonIcon type="previous" />}
```

Comment:
I was thinking the same thing. This button should have `aria-label="Show previous page"` and the other button should have `aria-label-"Show next page"`.